src/main/java/org/broadinstitute/consent/http/resources/NihAccountResource.java

```diff
@@ -32,8 +35,9 @@ public NihAccountResource(NihService nihService, UserService userService) {
     @RolesAllowed(RESEARCHER)
     public Response registerResearcher(NIHUserAccount nihAccount, @Auth AuthUser user) {
         try {
-            userService.findUserByEmail(user.getName());
-            return Response.ok(nihService.authenticateNih(nihAccount, user)).build();
+            User u = userService.findUserByEmail(user.getName());
+            List<UserProperty> authUserProps = nihService.authenticateNih(nihAccount, user, u.getDacUserId());
+            return Response.ok(authUserProps).build();
```

Comment:
nit, not your original code, but we could name things a bit better. `@Auth AuthUser user` should be `@Auth AuthUser authUser` and then `User u = userService.findUserByEmail(user.getName());` could be `User user = userService.findUserByEmail(user.getName());`

Reply:
Also not your code, but if we're here, we should fix it. Lines 24-25:
```
    private NihService nihService;
    private UserService userService;
```
should be 
```
    private final NihService nihService;
    private final UserService userService;
```